.github/workflows/build.yml

```diff
@@ -183,9 +183,10 @@ jobs:
       if: always()
       run: |
         mkdir -p build-history-data-new/cht-core/allure/${{ matrix.cmd }}-${{ matrix.suite }}/history
-        cp -r allure-report/history build-history-data-new/cht-core/allure/${{ matrix.cmd }}-${{ matrix.suite }}allure-report | true
+        cp -r allure-report/history build-history-data-new/cht-core/allure/${{ matrix.cmd }}-${{ matrix.suite }}/allure-report | true
```

Comment:
This is an error that was introduced when we split apart the e2e test suites. With this `/` missing the history gets lost. This was failing silently because of the `| true` on the end which is there to ignore the error for new suites without a history.

Reply:
Good catch